src/index.js

```diff
@@ -28,10 +28,13 @@ export default class I18n {
   }
 
   /**
-   * Leverages `numeral` for number formatting
+   * Leverages Intl.NumberFormat for currency formatting
    */
-  formatNumber (format: string, number: number): string {
-    return numeral(number).format(format)
+  formatNumber (number: number, style?: string, currency?: string): string {
+    const params = {}
+    if (style) params.style = style
+    if (currency) params.currency = currency
+    return new global.Intl.NumberFormat(this.locale, { style: style, currency: currency }).format(number)
```

Comment:
Support for this part of Intl spec looks pretty good in all browsers. Anyway just wondering if we should ship this package with an Intl pollyfill. 
Opinions @fcsonline @masylum ?

Reply:
Intl polyfill is a drama. This is supported for evergreen and IE11+ browsers so I wouldn't complicate it.